func.go

```diff
@@ -2065,7 +2068,7 @@ func compileRegexp(re, flags string) (*regexp.Regexp, error) {
 	}
 	r, err := regexp.Compile(re)
 	if err != nil {
-		return nil, fmt.Errorf("invalid regular expression %q: %s", re, err)
+		return nil, fmt.Errorf("invalid regular expression %q: %w", re, err)
```

Comment:
This change is find, but I don't think gojq library users unwrap the execution errors though.